Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.py

```diff
@@ -0,0 +1,2734 @@
+                                   'Both "device_id" and "sha2" arguments is required for endpoint delete action')
+    elif command == 'symantec-edr-endpoint-isolate':
+        action_type = 'Isolate Endpoint'
+        raw_response = client.get_isolate_endpoint(device_uid)
+    elif command == 'symantec-edr-endpoint-rejoin':
+        action_type = 'Rejoin Endpoint'
+        raw_response = client.get_rejoin_endpoint(device_uid)
+    else:
+        raise DemistoException('Endpoint Command action not found.')
+
+    title = f'Command {action_type}'
+
+    summary_data = {
+        "Message": raw_response.get('message'),
+        "CommandId": raw_response.get('command_id')
+    }
+
+    headers = list(summary_data.keys())
+    return CommandResults(
+        outputs_prefix=f'{INTEGRATION_CONTEXT_NAME}.Command.{action_type}',
+        outputs_key_field='command_id',
+        outputs=raw_response,
+        readable_output=tableToMarkdown(title, summary_data, headers=headers, removeNull=True),
+        raw_response=raw_response,
+        ignore_auto_extract=True
+    )
+
+
+def get_endpoint_status_command(client: Client, args: dict[str, Any]) -> CommandResults:
+    """
+    Check the command status of isolate endpoint.
+    Args:
+        client: client object to use.
+        args: all command arguments, usually passed from ``demisto.args()``.
+    Returns:
+        CommandResults: A ``CommandResults`` object that is then passed to ``return_results``, that contains an updated
+            result.
+    """
+    command_id = args.get('command_id', '')
+    readable_data = []
+    payload = post_request_body(args)
+    payload.pop('offset', 0)
+
+    raw_response = client.get_status_endpoint(command_id, payload)
+
+    summary_data = {
+        "state": raw_response.get('state'),
+        "Command Issuer Name": raw_response.get('command_issuer_name'),
+    }
+
+    if result := raw_response.get('status', ()):
+        for status in result:
+            summary_data['state'] = status.get('state', '')
+            summary_data['message'] = status.get('message', '')
+            summary_data['error_code'] = status.get('error_code', '')
+
+    if summary_data:
+        title = "Command Status"
+        readable_data.append(summary_data)
+        readable_output = generic_readable_output(readable_data, title)
+    else:
+        readable_output = 'No command status data to present.'
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix=f'{INTEGRATION_CONTEXT_NAME}.CommandStatus',
+        outputs_key_field='',
+        outputs=summary_data,
+        raw_response=raw_response,
+        ignore_auto_extract=True
+    )
+
+
+''' FETCHES INCIDENTS '''
+
+
+def fetch_incidents(client: Client) -> list:
+    """
+    Arguments Validation.
+    Args:
+        client: Client Object
+    Returns:
+        Incidents List
+    """
+
+    rev_incident_priority = {v: k for k, v in INCIDENT_PRIORITY_LEVEL.items()}
```

Comment:
why duplicate it? just use `INCIDENT_PRIORITY_LEVEL`

Reply:
Basically, it is required due to reverse mapping. Depending on fetch_priority user input e.g. High, Low, Medium need to create the query condition and pass it as a payload. Same is also applicable for Incident Status. Let me know if you have any different look out.